ecosystem/sep-0006.md

```diff
@@ -628,7 +836,9 @@ An anchor should also indicate in the `/info` response if they support the `fee`
 
 ## Fee
 
-The fee endpoint allows an anchor to report the fee that would be charged for a given deposit or withdraw operation. This is important to allow an anchor to accurately report fees to a user even when the fee schedule is complex. If a fee can be fully expressed with the `fee_fixed` and `fee_percent` fields in the `/info` response, then an anchor should not implement this endpoint.
+The fee endpoint allows an anchor to report the fee that would be charged for a given `/deposit` or `/withdraw` operation between equivalent assets. For non-equivalent assets the fees are included in the conversion price provided with [`SEP-38 GET /price`](sep-0038.md#get-price) or [`SEP-38 POST /quote`](sep-0038.md#post-quote).
```

Comment:
💡 I think this paragraph needs breaking up. Also, the paragraph talks about equivalent and non-equivalent assets and how fees apply to them, but I think we can make this easier to understand by saying which endpoints fees apply.

```suggestion
The fee endpoint allows an anchor to report the fee that would be charged when using the `/deposit` or `/withdraw` endpoints.

When using the `/deposit-exchange` or `/withdraw-exchange` endpoints fees are included in the conversion price provided with [`SEP-38 GET /price`](sep-0038.md#get-price) or [`SEP-38 POST /quote`](sep-0038.md#post-quote).
```

Reply:
https://github.com/stellar/stellar-protocol/pull/973/commits/7283bfcc3c38b3d24185becc32543a4dc83ca4de